zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -228,17 +235,19 @@ where
             "WITH eligible AS (
                  SELECT
                      {table_prefix}_received_notes.id AS id, txid, {output_index_col},
-                     diversifier, value, {note_reconstruction_cols}, commitment_tree_position,
+                     diversifier, value, {note_reconstruction_cols}, commitment_tree_position, 
                      SUM(value) OVER (ROWS UNBOUNDED PRECEDING) AS so_far,
-                     accounts.ufvk as ufvk, recipient_key_scope
+                     accounts.ufvk as ufvk, recipient_key_scope,
+                     transactions.block AS mined_height
```

Comment:
Using `transactions.block` here is correct because we need to have scanned the block (we don't know about it just from having observed a transparent UTXO, for example.)